kyaml/Makefile

```diff
@@ -39,6 +39,18 @@ vet:
 # likely related to https://github.com/kubernetes/kubernetes/issues/39188
 openapi:
 	sed -i 's/"x-kubernetes-patch-merge-key": "containerPort"/"x-kubernetes-patch-merge-key": "name"/g' 'openapi/kubernetesapi/swagger.json'
-	(which $(GOPATH)/bin/go-bindata || go get -v github.com/go-bindata/go-bindata)
+	(which $(GOPATH)/bin/go-bindata || go get -u github.com/go-bindata/go-bindata/...)
 	$(GOPATH)/bin/go-bindata --pkg kubernetesapi -o openapi/kubernetesapi/swagger.go openapi/kubernetesapi/swagger.json
 	$(GOPATH)/bin/go-bindata --pkg kustomizationapi -o openapi/kustomizationapi/swagger.go openapi/kustomizationapi/swagger.json
+
+VERSION="v1.19.1"
+schema:
+	which $(GOPATH)/bin/kind || GO111MODULE=on go get sigs.k8s.io/kind@v0.9.0
+	which $(GOPATH)/bin/kpt || GO111MODULE=on go get -v github.com/GoogleContainerTools/kpt
```

Comment:
These should be make targets, e.g.

https://github.com/kubernetes-sigs/kustomize/blob/master/Makefile#L315

Reply:
(then "schema" depends on them)